.github/workflows/cd_push.yml

```diff
@@ -6,9 +6,13 @@ on:
       image:
         required: true
         type: string
-      argocd_app_name:
+      argocd_app_names:
         required: true
         type: string
+      deploy_dirs:
```

Comment:
Why would we need multiple `deploy` dirs. 

Can you gave a example of how the file structure would look with this change

Reply:
i think this will work fine too:

global service: 
```
deploys/
  base/
  overlays/
    euc1-testing/
    euc1-staging/
    euc1/
    euw1/
 ```
 
sharded service: 
```
deploys/
  base/
  overlays/
    euc1-testing/
    euc1-staging/
    euc1-shared01/
    euc1-shared02/
    euw1-agents01/
 ```

Reply:
multiple deploy dirs are for multiple apps in the same cluster
```
deploy-app1/
  base/
  overlays/
    euc1-testing/
deploy-app2/
  base/
  overlays/
    euc1-testing/
```